commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/events/SubmissionRequestMessage.kt

```diff
@@ -13,6 +13,9 @@ data class SubmissionRequestMessage(
     @JsonProperty("fileMode")
     val fileMode: FileMode,
 
+    @JsonProperty("owner")
+    val submitter: String,
```

Comment:
I think this is consufusing, in submission system we have the concept of submitter and owner and here you are basically inverting it.

Reply:
You're right, it should be the owner. Nice catch.